app/scripts/metamask-controller.js

```diff
@@ -1848,6 +1853,39 @@ export default class MetamaskController extends EventEmitter {
 
   ///: BEGIN:ONLY_INCLUDE_IN(snaps)
 
+  /**
+   * Tracks snaps export usage. Note: This function is throttled to 1 call per 60 seconds.
+   *
+   * @param {string} handler - The handler to trigger on the snap for the request.
+   */
+  _trackSnapExportUsage = throttle(
+    (handler) =>
+      this.metaMetricsController.trackEvent({
+        event: 'Snap Export Used',
```

Comment:
This should probably be one of the `enum` values from `shared/constants/metametrics.ts` => `MetaMetricsEventName`.

Reply:
Done!